### ISSUE TITLE
Use stylelint-config-standard as a starting point?

### ISSUE THREAD

**Issue by keithjgrant** [ben-eb/perfectionist#81]
[stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard) provides a fairly standard set of opinions. This seems to hit a lot of the smaller, nit-picky rules. If we can agree on this with maybe some exceptions, we might be able to save a lot of needless discussion.

You can see its exact set of rules here: https://github.com/stylelint/stylelint-config-standard/blob/master/index.js

**Comment by keithjgrant** [comment]
One thing I do find odd in this config is a required empty line before a comment. I would do away with that.

It also requires an empty line between rules and between nested rules. Personally, I like that, but I'm sure others disagree.

**Comment by jeddy3** [comment]
> If we can agree on this with maybe some exceptions, we might be able to save a lot of needless discussion.

Yes, some legwork went into digging for the common rules found within a handful of CSS styleguides.

**Comment by jitendravyas** [comment]
I think there should be a separate stylelint standard config for formatting rules only. formatting is opinionated so I would like to have my own rules.
